Nexus.Sdk.Token/Responses/TokenOperationResponse.cs

```diff
@@ -5,7 +5,7 @@ namespace Nexus.Sdk.Token.Responses
     public record TokenOperationResponse
     {
         [JsonConstructor]
-        public TokenOperationResponse(string code, string hash, OperationAccountResponses senderAccount, OperationAccountResponses receiverAccount, decimal amount, string created, string? finished, string status, string type, string? memo, string? message, string cryptoCode, string tokenCode, string paymentReference)
+        public TokenOperationResponse(string code, string hash, OperationAccountResponses senderAccount, OperationAccountResponses receiverAccount, decimal amount, string created, string? finished, string status, string type, string? memo, string? message, string cryptoCode, string tokenCode, string paymentReference, decimal fiatAmount, decimal netFiatAmount)
```

Comment:
Should be done here as well

Reply:
Sorry for missing this one [Make fiat values optional](https://github.com/QuantozTechnology/Nexus.Sdk.Dotnet/pull/52/commits/ce68696e0d32efdadb57419cd8390fee9c34ca62)